session.opam

```diff
@@ -8,7 +8,9 @@ doc: "https://inhabitedtype.github.io/ocaml-session/"
 depends: [
   "ocaml" {>= "4.03.0"}
   "dune" {build & >= "1.0"}
-  "nocrypto"
+  "mirage-crypto"
+  "mirage-crypto-rng"
+  "base64"
   "result"
```

Comment:
this seems to be superfluous now (since 4.03.0 the `type ('a, 'b) result` is part of Stdlib/Pervasives).